A6lib.cpp

```diff
@@ -1,319 +1,368 @@
-#include <Arduino.h>
```

Comment:
Does removing these imports influence anything? What board are you running this on?

Reply:
Not actually.
I just removed it in other upcoming commits
I did all test with `Nodemcu` on my desk